README.md

```diff
@@ -1,3 +1,107 @@
 # safe-react-components
 
-[Live example](https://components.gnosis-safe.io/)
\ No newline at end of file
+![Badge](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg) 
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/gnosis/safe-react-components?sort=semver)
+![GitHub](https://img.shields.io/github/license/gnosis/safe-react-components)
+
+
+This repository contains a set of React components written in typescript.
+
+These components are being used to build [Safe-multisig](https://github.com/gnosis/safe-react) app.
+
+As Safe-multisig allows to integrate third party applications, Gnosis team recommends to build these apps using this components to get the following benefits:
+
+- Same style as the one used Safe-multisig. Making third-party apps looks native.
+- As the space used to render third-party apps is reduced, most of the components will be optimized to render properly in different resolutions.
+- Blockchain oriented: Some components will allow you to solve common problems like, inputs for ETH addresses and bigNumbers, Identicon images, etc.
+- Safe time, as you don't have to start from scratch building your safe-app.
+
+
+## How to install
+
+```bash
+   yarn add @gnosis.pm/safe-react-components
+```
+
+## Integration
+
+This library make use of [material-ui](https://material-ui.com/) as a peerDependency, it means you must install it in your Safe-app. Make sure to provide the same version as the one being use by the current version of this library.
+
+Once everything is installed, you have to instantiate a [ThemeProvider](https://styled-components.com/docs/api#themeprovider) from [styled-components](https://@gnosis.pm/safe-react-components/).
+
+This example uses the theme exported by safe-react-components. Here, you could extends this theme to customize it to your needs.
+
+```js
+import { ThemeProvider } from 'styled-components';
+import { theme } from '@gnosis.pm/safe-react-components';
+
+import App from './App';
+
+export default () => (
+  <ThemeProvider theme={theme}>
+    <App />
+  </ThemeProvider>
+);
+```
+
+### Using the same fonts as Safe-multisig
+
+If you want your app has the same fonts as the one used by Safe-multisig you need to do next.
+
+```js
+import { createGlobalStyle } from 'styled-components';
+import avertaFont from '@gnosis.pm/safe-react-components/dist/fonts/averta-normal.woff2';
+import avertaBoldFont from '@gnosis.pm/safe-react-components/dist/fonts/averta-bold.woff2';
+
+const GlobalStyle = createGlobalStyle`
+    @font-face {
+        font-family: 'Averta';
+        src: local('Averta'), local('Averta Bold'),
+        url(${avertaFont}) format('woff2'),
+        url(${avertaBoldFont}) format('woff');
+    }
+`;
+
+export default GlobalStyle;
+```
+
+And then include it wrapping your application.
```

Comment:
Not sure about the _wrapping_ as it's not wrapping it, but at the same `App` level.